x-pack/plugin/otel-data/README.md

```diff
@@ -0,0 +1,44 @@
+NOTE: this plugin is not related to APM Metrics used in ES codebase. The APM Metrics are in :modules:apm
+
+## OpenTelemetry Ingest plugin
+
+The OpenTelemetry Ingest plugin installs index templates and component templates for Elastic OpenTelemetry data.
+
+All resources are defined as YAML under [src/main/resources](src/main/resources).
+
+The OpenTelemetry index templates rely on mappings from `x-pack-core`.
+See [x-pack/plugin/core/src/main/resources](../core/src/main/resources).
+
+## Adding/Removing/Updating a resource
+
+All resources are defined as YAML under [src/main/resources](src/main/resources).
+
+For a resource to be known to the plugin it must be added to
+[src/main/resources/resources.yaml](src/main/resources/resources.yaml) in the
+appropriate section.
+
+Any update to resources included by this package also requires a bump to the
+`version` property included in the resources file.
+
+## Testing
+
+## Unit testing
+
+Java unit tests cover basic, low-level details of the plugin, such as the parsing and loading of resources.
+These can be run with:
+
+```
+./gradlew x-pack:plugin:apm-data:test
```

Comment:
```suggestion
./gradlew :x-pack:plugin:apm-data:test
```

Reply:
Thanks, fixed this - also this should have been `:x-pack:plugin:otel-data:test`, fixed that as well.